public/index.html

```diff
@@ -17,28 +17,38 @@
       <%= VUE_APP_TIER %>
         <%= VUE_APP_TITLE %>
     </title>
-    <meta name="title" content="70 years of hydrological drought">
+    <meta name="title" content="Five droughts that changed history">
     <meta name="description" content="">
     <!-- Open Graph / Facebook -->
     <meta property="og:type" content="website">
     <meta property="og:url" content="https://labs.waterdata.usgs.gov/visualizations/gw-conditions/index.html#/">
-    <meta property="og:title" content="70 years of hydrological drought">
+    <meta property="og:title" content="Five droughts that changed history">
     <meta property="og:description" content="">
-    <meta property="og:image" content="https://labs.waterdata.usgs.gov/visualizations/thumbnails/gw-conditions-og-peaks.png">
+    <meta property="og:image" content="https://labs.waterdata.usgs.gov/visualizations/images/drought_timeline_thumbnail.png">
     <!-- Twitter -->
     <meta property="twitter:card" content="summary_large_image">
     <meta property="twitter:url" content="https://labs.waterdata.usgs.gov/visualizations/gw-conditions/index.html#/">
```

Comment:
I think there could be slightly bigger margins on this thumbnail, with a slight decrease in the text spacing to make room